### ISSUE TITLE
Add a setting to change cursor animation duration / speed

### ISSUE THREAD

**Issue by Oneechan69** [neovide/neovide#2886]
**Is your feature request related to a problem? Please describe.**
I like the cursor animation when I do `^`and `$` but I would like the duration of the animation to be faster (like 0.15 secs rather than 0.3 secs; duration, not cursor animation length). This should be a config option for this.

**Comment by cpmsmith** [comment]
I think the [`neovide_cursor_animation_length` option](https://neovide.dev/configuration.html#animation-length) is in fact what you're looking for – it does change the speed of the animation.

**Comment by Oneechan69** [comment]
Oh I thought that documentation was a mistake. I didn't see much difference when playing around low values, they appeared to change the physical length of the animation (how I interpreted the word length), I only noticed when I set the value 1. It would be more intuitive if the setting was called `neovide_cursor_animation_duration` rather than `_length`. I didn't notice the setting right below it saying [Animation Trail Size](https://neovide.dev/configuration.html#animation-trail-size) which was the setting I was looking for and mistook it for. With that in mind I created two new issues:

- https://github.com/neovide/neovide/issues/2894
- https://github.com/neovide/neovide/issues/2895
